panoptes_client/avatar.py

```diff
@@ -0,0 +1,31 @@
+from panoptes_client.panoptes import (
+    Panoptes,
+    PanoptesObject,
+    LinkResolver,
+)
+from panoptes_client.project import Project
+
+
+class Avatar(PanoptesObject):
```

Comment:
Let's rename this to `ProjectAvatar` to make it clear that it's project specific. Or, maybe move it into `project.py`.

Reply:
Renamed, but happy to move it into project.py if that is preferred!